Server/Core/Commands/SurveillanceHandler.cs

```diff
@@ -76,12 +77,13 @@ public static void HandleGetProcessesResponse(Client client, GetProcessesRespons
 
         public static void HandleGetKeyloggerLogsResponse(Client client, GetKeyloggerLogsResponse packet)
         {
-            if (client.Value.FrmKl == null)
+            if (client.Value == null || client.Value.FrmKl == null)
                 return;
 
             if (packet.FileCount == 0)
             {
-                client.Value.FrmKl.SetGetLogsEnabled(true);
+                if (client.Value != null && client.Value.FrmKl != null)
```

Comment:
I know we are handling these exceptions in the methods, but for the static methods that are threaded shouldn't we need to check if Value is nulled before calling one of these methods?